analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/InitializationVectorShouldBeRandom.CSharp11.cs

```diff
@@ -12,7 +12,7 @@ public void SymetricAlgorithmCreateEncryptor()
         {
             sa.GenerateKey();
             var generateIVNotCalled = sa.CreateEncryptor(sa.Key, sa.IV);
-            var constantVector2 = sa.CreateEncryptor(sa.Key, "1234567890123456"u8.ToArray()); // FIXME Non-compliant
+            var constantVector2 = sa.CreateEncryptor(sa.Key, "1234567890123456"u8.ToArray()); // FN
```

Comment:
What do you think of just flagging all calls to `ToArray()` as `Weak`?

Reply:
Sounds good but I'll go it as an FN PR after this one to ensure the delivery.

Reply:
This might cause some FPs. Let's be careful.

Reply:
We'll see in the other PR and we can revert after validation.

Reply:
for the record - there's already an issue for this in our backlog.

https://github.com/SonarSource/sonar-dotnet/issues/4565